azure/image/rhel/azure-rhel.pkr.hcl

```diff
@@ -22,11 +31,11 @@ source "azure-arm" "rhel" {
   build_resource_group_name         = "${var.resource_group}"
 
   shared_image_gallery_destination {
-    subscription         = "${var.subscription_id}"
-    resource_group       = "${var.resource_group}"
-    gallery_name         = "${var.az_gallery_name}"
-    image_name           = "${var.az_gallery_image_name}"
-    image_version        = "${var.az_gallery_image_version}"
+    subscription   = "${var.subscription_id}"
+    resource_group = "${var.resource_group}"
+    gallery_name   = "${var.az_gallery_name}"
+    image_name     = "${var.az_gallery_image_name}"
+    image_version  = "${var.az_gallery_image_version}"
```

Comment:
nit: can these formatting changes be avoided ? Or is there a way to make it consistent and enforce it ?

Reply:
I think this can be enforced by using some of `terraform fmt` script in CI. Created an issue for this: https://github.com/confidential-containers/cloud-api-adaptor/issues/1487